packages/repository/src/__tests__/unit/repositories/legacy-juggler-bridge.unit.ts

```diff
@@ -419,6 +618,21 @@ describe('DefaultCrudRepository', () => {
       'execute() must be implemented by the connector',
     );
   });
+
+  it('has the property inclusionResolvers', () => {
+    const repo = new DefaultCrudRepository(Note, ds);
+    expect(repo.inclusionResolvers).to.not.be.undefined();
```

Comment:
Let's improve this assertion and check that `inclusinoResolvers` has the right type/structure. Something along the following lines:

```suggestion
    expect(repo.inclusionResolvers).to.be.instanceof(Map);
```

Reply:
will it better if we add [hasInclusionResolvers ](https://github.com/strongloop/loopback-next/pull/3387/files#diff-785622a1e0ea38ec20dc8e91adbf6534R226)?